tweets/urls.py

```diff
@@ -1,12 +1,13 @@
 from django.urls import path
-from . import views
-
+from . import views as tweets_views
+from .retweets import views as retweets_views
 app_name = 'tweets'
 
 urlpatterns = [
-  path('create/', views.TweetsViews.as_view(), {'action': 'create'}, name='createTweet'),
-  path('show/<int:id>/', views.TweetsViews.as_view(), {'action': 'show'}, name='showTweet'),
-  path('edit/<int:id>/', views.TweetsViews.as_view(), {'action': 'edit'}, name='editTweet'),
-  path('update/<int:id>/', views.TweetsViews.as_view(), {'action': 'update'}, name='updateTweet'),
-  path('destroy/<int:id>/', views.TweetsViews.as_view(), {'action': 'destroy'}, name='destroyTweet'),
+  path('create/', tweets_views.TweetsViews.as_view(), {'action': 'create'}, name='createTweet'),
+  path('show/<int:id>/', tweets_views.TweetsViews.as_view(), {'action': 'show'}, name='showTweet'),
+  path('edit/<int:id>/', tweets_views.TweetsViews.as_view(), {'action': 'edit'}, name='editTweet'),
+  path('update/<int:id>/', tweets_views.TweetsViews.as_view(), {'action': 'update'}, name='updateTweet'),
+  path('destroy/<int:id>/', tweets_views.TweetsViews.as_view(), {'action': 'destroy'}, name='destroyTweet'),
+  path('<int:original_tweet_id>/retweet/', retweets_views.RetweetView.as_view(http_method_names=['post']), name='reTweet'),
```

Comment:
Why create is using a different syntax for POST and Retweet Uses a different syntax. 

They both are form submissions

Reply:
didn't get that as create is for new tweet and retweet require an Id